<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpDialog</class>
 <widget class="QDialog" name="TcpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tcpView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>651</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrws1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>351</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UMing CN</family>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrws2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UMing CN</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="tcpSave">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="canc">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>430</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
